### PR TITLE
Imorove Tile3DLayer.isLoaded

### PR DESCRIPTION
**Pull request by chrisgervang**
For #8260

#### Change List
- Add protections to Tile3DLayer.isLoaded

### CODE REVIEW

**Comment by chrisgervang** [comment]
Should I open a second PR for v9 since that file is "disabled"?

File: modules/geo-layers/src/tile-3d-layer/tile-3d-layer.ts [hunk 1]
```diff
@@ -105,8 +105,12 @@ export default class Tile3DLayer<DataT = any, ExtraPropsT extends {} = {}> exten
   }
 
   get isLoaded(): boolean {
-    const {tileset3d} = this.state;
-    return tileset3d !== null && tileset3d.isLoaded();
+    return Boolean(
```
Comment (felixpalmer) [inline modules/geo-layers/src/tile-3d-layer/tile-3d-layer.ts:108]:
```suggestion
    return Boolean(this.state?.tileset3d?.isLoaded() && super.isLoaded)
```

Like we do in `TileLayer`

**Review by felixpalmer — CHANGES_REQUESTED** [review]

**Comment by felixpalmer** [comment]
> Should I open a second PR for v9 since that file is "disabled"?

That would be great

**Comment by chrisgervang** [comment]
@felixpalmer good to go?

**Review by felixpalmer — APPROVED** [review]

**Pull request merged**